packages/victory-box-plot/src/helper-methods.js

```diff
@@ -206,12 +206,23 @@ const getCalculatedValues = (props) => {
 
 // eslint-disable-next-line complexity
 const getWhiskerProps = (props, type) => {
-  const { horizontal, style, boxWidth, whiskerWidth, datum, scale, index } = props;
+  const {
+    ariaLabel,
```

Comment:
`ariaLabel` and `tabIndex` props shouldn't be passed from `VictoryBoxPlot` to its primitives. Only the primitives need to know about these props. `VictoryBoxPlot`, `VictoryBar` etc shouldn't use them